### PR TITLE
change CI registry hostname

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

File: utils.sh [hunk 1]
```diff
@@ -432,10 +432,10 @@ function write_pull_secret() {
 
     verify_pull_secret
 
-    # Get a current pull secret for registry.svc.ci.openshift.org using the token
+    # Get a current pull secret for registry.ci.openshift.org using the token
     tmpkubeconfig=$(mktemp --tmpdir "kubeconfig--XXXXXXXXXX")
     _tmpfiles="$_tmpfiles $tmpkubeconfig"
-    oc login https://api.ci.openshift.org --kubeconfig=$tmpkubeconfig --token=${CI_TOKEN}
+    oc login https://api.ci.l2s4.p1.openshiftapps.com:6443 --kubeconfig=$tmpkubeconfig --token=${CI_TOKEN}
```
Comment (honza) [inline utils.sh:438]: This URL seems unstable.

**Comment by honza** [comment]
Works for me

**Comment by dhellmann** [comment]
@stbenjam @honza I've updated this PR to include some docs and the Dockerfile that pointed to the old registry location

**Comment by stbenjam** [comment]
/lgtm

Thanks for fixing this!

**Comment by danwinship** [comment]
I think this makes it impossible to deploy a cluster from a cluster-bot build:

```
+(ocp_install_env.sh:16): extract_command(): oc adm release extract --registry-config /opt/dev-scripts/pull_secret.json --command=oc --to /tmp/installer--xCrHn4fJc2 registry.svc.ci.openshift.org/ci-ln-ds9x5ht/release:latest
error: unable to read image registry.svc.ci.openshift.org/ci-ln-ds9x5ht/release:latest: Head "https://registry.svc.ci.openshift.org/v2/ci-ln-ds9x5ht/release/manifests/latest": unauthorized: authentication required
```

**Comment by cybertron** [comment]
It also broke the local image build functionality, at least for some repos. For example, MCO uses an image from the old registry as their base image, so now that we're logging in to a different registry the image build no longer has permission to pull it.

I'm not sure what the solution is. Are all image references supposed to be updated to the new registry? If so, that's probably a discussion we need to have with the broader OpenShift team.

**Comment by dhellmann** [comment]
We have to at least use the new one because that's where the API for determining the current CI build points now. I was told the registries will mirror the same content, for the time being. Perhaps we should add a second token variable and login to both registries to support the image build case, until all of the relevant Dockerfiles are updated?

**Comment by cybertron** [comment]
Can you log in to two different clusters at once? I tried logging in to the old one before running dev-scripts and it didn't help.

**Comment by dhellmann** [comment]
> Can you log in to two different clusters at once? I tried logging in to the old one before running dev-scripts and it didn't help.

We use `oc login registry` to generate a pull-secret file. Then we merge several files together to make the one actually used. So, we could run `oc login registry` twice, save the output to separate files, then add both to the list of files we merge.

**Comment by cybertron** [comment]
Looks like the openshift-bot is proposing changes to update the image references in Dockerfiles, so this will probably resolve itself soon.

For reference: https://github.com/openshift/baremetal-runtimecfg/pull/115/files

**Comment by danwinship** [comment]
Yeah, I think cluster-bot got fixed a few days after I commented above. OCP is just in a temporarily inconsistent state right now...

**Pull request merged**